codestodians/komlie-frontend
# [Feature] Create product details page

#### What does this PR do?
- Creates a product-details page

#### Description of Task to be completed?
- Create a product-details component with the accompanying route
- Activate navigation and display of available details on the product-details page.
- Abstract product component from product-list component.

#### How should this be manually tested?
- Run the application
- Click on the `cosmetics` link
- Click on any of the products
- You should be navigated to a page that contains details about only the product with the url showing the ID of the particular product.

#### What are the relevant pivotal tracker stories?
- [#166755827](https://www.pivotaltracker.com/story/show/166755827)

#### Screenshots (if appropriate)
![Screen Shot 2019-06-18 at 4 01 58 PM (2)](https://user-images.githubusercontent.com/22825700/59695673-84e8da80-91e2-11e9-87f1-22bacf2ee8a1.png)

#### Extra details
- When the product-details page is reloaded, there is a redirect to `products`. This is due to the fact that the product details is only passed from one route to the other as it stands. Shortly, a story is coming to write the data to cache so that data persistence is in place once redirected to a details page.
- The product-details page is minimalistic because my first goal is to make the page/component available and then work (and more details) can be done on it subsequently.
